Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -14511,6 +14511,116 @@ def pan_os_list_profile_exception_command(args: dict) -> CommandResults:
     )
 
 
+def build_master_key_create_or_update_cmd(args: dict, action: Literal['create', 'update']) -> str:
+    """Builds the XML command for creating or updating the default master key on Panorama / PAN-OS.
+
+    Args:
+        args (dict): The command arguments.
+        action ('create'| 'update'): Whether to create a new master key or update an existing one.
+
+    Returns:
+        str: XML string of the master key create or update command.
+    """
+    master_key_args = [
+        add_argument(arg=args.get('lifetime_in_hours'), field_name='lifetime', member=False),
+        add_argument(arg=args.get('reminder_in_hours'), field_name='reminder', member=False),
+    ]
+    if action == 'create':
+        master_key_args.append(
+            add_argument(arg=args.get('master_key'), field_name='new-master-key', member=False),
+        )
+    else:
+        master_key_args.extend(
+            [
+                add_argument(arg=args.get('new_master_key'), field_name='new-master-key', member=False),
+                add_argument(arg=args.get('current_master_key'), field_name='current-master-key', member=False),
+            ]
+        )
+
+    master_key_args.append(add_argument_yes_no(arg='no', field_name='on-hsm'))
+    master_key_element = add_argument(arg=''.join(master_key_args), field_name='master-key', member=False)
+
+    return add_argument(arg=master_key_element, field_name='request', member=False)
+
+
+def create_or_update_master_key(args: dict, action: Literal['create', 'update']) -> CommandResults:
+    """Builds an XML command and sends a request to create or update the master key based on the given action.
+
+    Args:
+        args (dict): The command arguments.
+        action ('create'| 'update'): Whether to create a new master key or update an existing one.
+
+    Returns:
+        CommandResults: Contains readable output and raw response.
+    """
+    master_key_cmd = build_master_key_create_or_update_cmd(args, action=action)
+    raw_response: dict = http_request(URL, 'GET', params={'type': 'op', 'key': API_KEY, 'cmd': master_key_cmd})
+    response_result = dict_safe_get(raw_response, ('response', 'result'))  # human readable message
+
+    # Creating or updating the encryption master key by definition invalidates the current API key, refer to the integration docs.
+    demisto.info(f'The master key of {URL} has been {action}d. The current API key has been invalidated.')
+
+    return CommandResults(
+        readable_output=f'{response_result}. The current API key has been invalidated. Generate a new API key and ensure the integration instance is updated accordingly.',
```

Comment:
```suggestion
        readable_output=f'{response_result}. ⚠️ The current API key is no longer valid! (by design) Generate a new API key and update it in the integration instance configuration to keep using the integration.',
```

Reply:
Updated human readable output in commit 77379b70697a17549d3313dd6b3d99ddcb0cc026